<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bus Stop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>52</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Bus 1</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>탑승인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="bus1Cnt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="btnBus1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>10번 버스</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Bus 2</string>
    </property>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>탑승인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="bus2Cnt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="btnBus2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>20번 버스</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Bus 3</string>
    </property>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>탑승인원</string>
     </property>
    </widget>
    <widget class="QLabel" name="bus3Cnt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="btnBus3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>30번 버스</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="busPlus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>탑승대기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="busMinus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>탑승취소</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
